src/components/ApplicationDetailsView/ApplicationDetailsView.scss

```diff
@@ -0,0 +1,6 @@
+.application-details {
```

Comment:
Maybe this needs to go into the `GettingStartedCard` component. Keep a default sizing for the image with this style that can be overridden with props.